trailblazer/cli/core.py

```diff
@@ -90,6 +90,17 @@ def cancel(context, analysis_id):
         LOG.error(e)
 
 
+@base.command("set-completed")
+@click.argument("analysis_id", type=int)
+@click.pass_context
+def set_analysis_completed(context, analysis_id):
+    """Set status of an analysis to "COMPLETED" """
+    try:
+        context.obj["trailblazer"].set_analysis_completed(analysis_id=analysis_id)
+    except Exception as e:
```

Comment:
What does "e" stand for. Feel a bit cryptic and unsafe to just call it "e".

Reply:
This has always been a standard thing to call error object. Ive been copying how its always been done in cg... maybe time to rethink